producer/s3.go

```diff
@@ -520,8 +520,6 @@ func (prod *S3) transformMessages(messages []*core.Message) {
 	bufferedMessages := []*core.Message{}
 	// Format and sort
 	for _, msg := range messages {
-		originalMsg := msg.Clone()
```

Comment:
merge conflict because of #173 - reverting this file should resolve the merge conflict